photon-ml/src/main/scala/com/linkedin/photon/ml/cli/game/scoring/Params.scala

```diff
@@ -60,49 +81,57 @@ object Params {
     val parser = new OptionParser[Params]("GLMix-Scoring-Params") {
       opt[String]("input-data-dirs")
         .required()
-        .text("Input directories of data used to compute the viewer-actor-activityType affinity score. " +
-        "Multiple input directories are separated by commas.")
+        .text("Input directories of data to be scored. Multiple input directories are also accepted if they are " +
+          "separated by commas, e.g., inputDir1,inputDir2,inputDir3.")
         .action((x, c) => c.copy(inputDirs = x.split(",")))
       opt[String]("date-range")
-        .text("Date range for the input data represented in the form start.date-end.date, e.g. 20150501-20150631, " +
-        s"default: ${defaultParams.dateRangeOpt}.")
+          .text(s"Date range for the input data represented in the form start.date-end.date, " +
+              s"e.g. 20150501-20150631. If this parameter is specified, the input directory is expected to be in the " +
+              s"daily format structure (e.g., inputDir/daily/2015/05/20/input-data-files). Otherwise, the input paths " +
+              s"are assumed to be flat directories of input files (e.g., inputDir/input-data-files). " +
+              s"Default: ${defaultParams.dateRangeOpt}.")
+          .action((x, c) => c.copy(dateRangeOpt = Some(x)))
+      opt[String]("date-range-days-ago")
+          .text(s"Date range for the input data represented in the form start.daysAgo-end.daysAgo, " +
+              s"e.g. 90-1. If this parameter is specified, the input directory is expected to be in the " +
+              s"daily format structure (e.g., inputDir/daily/2015/05/20/input-data-files). Otherwise, the input paths " +
+              s"are assumed to be flat directories of input files (e.g., inputDir/input-data-files). " +
+              s"Default: ${defaultParams.dateRangeDaysAgoOpt}.")
+          .action((x, c) => c.copy(dateRangeDaysAgoOpt = Some(x)))
         .action((x, c) => c.copy(dateRangeOpt = Some(x)))
       opt[String]("feature-name-and-term-set-path")
           .required()
-          .text("Input path to the features name-and-term lists.")
+          .text(s"Input path to the features name-and-term lists.")
```

Comment:
Are the 's' characters necessary at the start of the strings even if no string interpolation is happening?